### ISSUE TITLE
Enhancement Request for Support of the Geoshape Field

### ISSUE THREAD

**Issue by jojo3310** [Esri/Survey123Community#160]
I notice that currently the Geoshape feature is not ready to be released.  I have a Survey123 application that would benefit our organization greatly if that feature was available.  I would like to request that emphasis be put on the Geoshape feature so that we and others can benefit from this application more fully.

**Comment by mdayfcnsw** [comment]
I'd like to +1 for the enhancement of the Geoshape feature. Lines and polygons would be a huge advantage for us and a great supplement to points!
Thanks for listening!!

**Comment by ljaupi** [comment]
+1 Lines and polygons are a must.

**Comment by esauthier** [comment]
+1, Polygons and Line are a necessity!

**Comment by travisbutcher** [comment]
:+1: Points, Lines and Polygons are essential to GIS Data.

**Comment by HendrikBritz** [comment]
+1

**Comment by veddha** [comment]
+1!! 

I understand this is the form-based solution, but still need ability to capture line/polygon geometries!!

**Comment by kimhastings** [comment]
Agreed. Points, lines and polygons are essential for representing spatial data.

**Comment by joparikh** [comment]
+1

**Comment by andreoudi** [comment]
I agree! Lines and polygons are necessary!!! When can you implement this? Any ideas?

**Comment by vtran3** [comment]
+1

**Comment by alangdon** [comment]
+1 for geotrace capability

**Comment by jullums** [comment]
+1

**Comment by ravi23677** [comment]
+1 for polylines and polygons

**Comment by nicsi76** [comment]
+1

**Comment by billingsmc** [comment]
+1 - for more geometries. Also, need a +1 button on this site. :)

**Comment by mjbullock** [comment]
+1 for Geoshape support.

**Comment by bstiber74** [comment]
+1 for geoshape and geotrace

**Comment by jdaviesmitcham** [comment]
A freehand drawn polygon on a map may be a good first stage with GPS for the centroid coordinate of the polygon, otherwise +1 for geoshape and geotrace

**Comment by DavidATait** [comment]
Can you indicate when capturing polygons will be added to the "planned" enhancements?  I have a number of customer projects that need limited polygon capture together with significant form capture.

**Comment by bcgisjosh** [comment]
+1 for adding lines and polygon functionality to Survey123

**Comment by HendrikBritz** [comment]
+1 for adding lines and polygon functionality to Survey123

**Comment by mhoole** [comment]
+1 for adding lines and polygon functionality to Survey123

**Comment by mstewart82** [comment]
+1 for adding lines and polygon functionality to Survey123

**Comment by nickoadl** [comment]
+1 for polygons and polylines in survey 123

**Comment by beutelt** [comment]
+1 for polygons in survey123

**Comment by bart-richardson** [comment]
+1 for polygons and lines in survey123

**Comment by GISfast** [comment]
+1 for polygons Our small historical society could do much more to identify/protects sites if map users could articulate areas (ex family cemeteries) using an ArcGIS Online mobile app!

**Comment by smjett** [comment]
+1

**Comment by bgkerr** [comment]
+1
